src/main/java/frc/robot/Operator.java

```diff
@@ -83,8 +92,13 @@ private void controlPanelControl(){
             resetControlPanel();
         }
         
+        //Pos control timeout
+        if((lastPositionPress + POSITION_TIMEOUT) < System.currentTimeMillis()) {
+            finishedPositionControl = true;
+        }
+        
         //Rotation control
-        if(OP.getLeftBumper()) {
+        if(OP.getLeftBumper() && ((lastButtonPress + BUTTON_DELAY) < System.currentTimeMillis())) {
```

Comment:
I assume the timing here is to prevent double clicks? You could just use boolean logic instead to make sure when they press the button once it is only processes as once.

Reply:
Timing is better for my method of implementation. I can explain more in slack if you want.